### PR TITLE
displaying message now when there are no breakpoints to show in the breakpoint panel

### PR DESCRIPTION
**Pull request by karina-y**
fixes issue #1077 
i followed the same format message in the comments panel for the empty breakpoints panel. the text will also wrap to fit the width of the panel and hide when the breakpoint panel is collapsed.

before:
image

after:
image

### CODE REVIEW

**Comment by jonbell-lot23** [comment]
Copy and visual looks good to me, thanks

**Comment by karina-y** [comment]
tests timing out in ci:
stepping-05, passes locally

**Comment by jonbell-lot23** [comment]
Another note on this, inspired by a chat I just had with Jason.

There are sometimes two modes for thinking about design: "good enough" versus "wouldn't it be cool if?" At Microsoft, we sometimes talked about gold/silver/bronze for the level of design support each feature would get.

So from a "good enough" / bronze level of support, this looks good and ready to go! But if I were thinking more broadly about this issue I'd be thinking about illustrations, loading states/edgecases, adding some whimsy to the copy, etc.

So that's something I'm thinking about. We might want a way to quantify the level of design support on tickets so we have a vocabulary for saying "Looks good! But let's open a separate ticket to make it even better later!"

**Comment by karina-y** [comment]
yeah i saw that chat, i liked the idea of using the space for more instructional opportunity. i suppose my thought process is to launch the bronze as the gold is being developed and designed, what are your thoughts @jonbell-lot23 and @jasonLaster ?

File: src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/index.js [hunk 1]
```diff
@@ -45,7 +45,11 @@ class Breakpoints extends Component {
     const { breakpointSources } = this.props;
 
     if (!breakpointSources.length) {
-      return null;
+      return (
+        <div className="no-breakpoints">
```
Comment (jaril) [inline src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/index.js:49]: Small nit, it's better to have classnames that describe the element. 

`no-breakpoints` mostly describes the state. For this, something like `empty-state-content` is more descriptive.

**Review by jaril — APPROVED** [review]

File: src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/index.js (hunk 1 above)
Comment (karina-y) [reply src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/index.js]: all done

**Comment by jonbell-lot23** [comment]
Closing the loop here -- yup, let's do the bronze level for now and make a new task for making this better, as part of the great onboarding experience work we want to do.

**Review by jasonLaster — APPROVED** [review]

**Comment by karina-y** [comment]
tests timing out in ci:
stepping-05 passes locally
object_preview-05 times out in master and branch

**Pull request merged**